payments_service/braintree/templates/braintree/emails/subscription_header.html

```diff
@@ -10,5 +10,6 @@
   <body id="email">
     <main>
       <div id="wrapper">
-        <div style="background-image: url(http://placeholder)" id="logo"></div>
+        <div id="logo"></div>
         <h1>{{ seller.name }}</h1>
+        <p class="user-email">placeholder@placeholder.com</p>
```

Comment:
Actually looking at this, its kinda wierd, why does an email to user have their email address in the top of it? cc @brampitoyo